stdlib/universe/holt_winters/holt_winters.go

```diff
@@ -262,7 +278,7 @@ func (r *HoltWinters) sse(params *mutable.Float64Array) float64 {
 			// Compute error
 			if math.IsNaN(fcast.Value(i)) {
 				// Penalize fcast NaNs
-				return math.Inf(1)
+				return math.MaxFloat64
 			}
```

Comment:
Should we just remove this check for IsNaN? And just let the NaNs continue to exist since they must have come from the input data if they do? This way we are not adding extra logic that we don't intend to actually use. If it triggers an error thats fine as the user can then find and fix the source of their own NaNs. This check as its stands will hide that fact from them and cause holt winters to optimize for the NaNs because of the overflow issue we found.

Reply:
That makes sense. Did a quick test with NaN in the input after removing the above `IsNaN` check. 
The code panics with a message - "`caught panic: optimize: initial function value is NaN`"
Shall I capture `NaN` case in the code and return a user error rather than letting it panic?